includes/Classifai/Plugin.php

```diff
@@ -188,6 +188,14 @@ public function enqueue_admin_assets() {
 				'ajax_nonce'   => wp_create_nonce( 'classifai' ),
 			]
 		);
+
+		wp_enqueue_script(
```

Comment:
Getting Missing Dependencies error with the current version of WP. Maybe we should enqueue the script conditionally only when the `wp-commands` script is registered? which will be registered in WP 6.3 as mentioned in the PR description.

```php
if ( wp_script_is( 'wp-commands', 'registered' ) ) {
   // Enqueue script
}
```

<img width="1003" alt="image" src="https://github.com/10up/classifai/assets/10613171/7d5a6f58-4030-483e-899c-41123ca701c9">

Reply:
Ah, nice catch. Forgot to test with the Gutenberg plugin disabled. This should be fixed now